packages/block-library/src/editor.scss

```diff
@@ -34,6 +34,11 @@
 @import "./verse/editor.scss";
 @import "./video/editor.scss";
 
+/**
+ * Import styles from internal editor components used by the blocks.
+ */
+@import "./block/edit-panel/style.scss";
```

Comment:
They should be also renamed to `editor.scss` if they belong to this group of styles.

Reply:
The rename was applied.